zerver/openapi/test_curl_examples.py

```diff
@@ -16,14 +16,12 @@
 
 from zerver.models import get_realm
 from zerver.openapi import markdown_extension
-from zerver.openapi.curl_param_value_generators import (
-    CALLED_GENERATOR_FUNCTIONS,
-    REGISTERED_GENERATOR_FUNCTIONS,
-)
+from zerver.openapi.curl_param_value_generators import AUTHENTICATION_LINE, functions_not_called
 
 
-def test_generated_curl_examples_for_success(client: Client, owner_client: Client) -> None:
+def test_generated_curl_examples_for_success(client: Client) -> None:
     authentication_line = f"{client.email}:{client.api_key}"
```

Comment:
Let's rename this to `DEFAULT_AUTHENTICATION_LINE`.

Reply:
Renamed `authentication_line` to `default_authentication_line`(in small letters because `default_authentication_line ` in not global variable). Also , added about this change in commit description.